### PR TITLE
Enable SYCL tests

### PR DESCRIPTION
**Pull request by masterleinad**
- I only fixed the places in `src/Kokkos_ArithTraits` that were necessary for running the test with the SYCL backend successfully. Using https://github.com/kokkos/kokkos/blob/develop/core/src/Kokkos_MathematicalFunctions.hpp would probably make most of it redundant.
- We use KOKKOS_IMPL_DO_NOT_USE_PRINTF internally to work around `printf` not being directly available with `SYCL`. The function is not supposed to be used outside the `Kokkos` since we would want to remove it at any time when there is proper compiler support. I nevertheless used it here. Let me know if you want the `printf`s to be disabled instead.
- Some failing tests are still disabled. I haven't looked too hard into it.

### CODE REVIEW

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pre-Test Inspection' - - This Pull Request Requires Inspection... The code must be inspected by a member of the Team before Testing/Merging
           WARNING: NO REVIEWERS HAVE BEEN REQUESTED FOR THIS PULL REQUEST!

**Comment by lucbv** [comment]
@masterleinad I feel sorry you had to do that, I am working on PR #906 to avoid creating all the `*.cpp` per backend.
Are you in a rush to have this merged in or can you wait a bit on the other PR? The refactor will require only one `.cpp` (or maybe one per component) which will be more manageable...

**Comment by masterleinad** [comment]
>  Are you in a rush to have this merged in or can you wait a bit on the other PR? The refactor will require only one `.cpp` (or maybe one per component) which will be more manageable...

No, no rush here. Actually, it wasn't too bad. Just copying the files from Cuda and doing some renaming via regexes. :slightly_smiling_face:

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pre-Test Inspection' - - This Pull Request Requires Inspection... The code must be inspected by a member of the Team before Testing/Merging
           NO REVIEWS HAVE BEEN PERFORMED ON THIS PULL REQUEST!

**Comment by lucbv** [comment]
@masterleinad the refactor of the unit-tests has been merged in develop and will lead require some re-write.
It should not take much time, although new pre-processor guards will be needed in some unit-tests headers...

**Comment by masterleinad** [comment]
@lucbv I rebased and removed the test files that are auto-generated now.

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pre-Test Inspection' - - This Pull Request Requires Inspection... The code must be inspected by a member of the Team before Testing/Merging
           NO REVIEWS HAVE BEEN PERFORMED ON THIS PULL REQUEST!

File: unit_test/CMakeLists.txt [hunk 1]
```diff
@@ -87,6 +87,58 @@ IF (KOKKOS_ENABLE_CUDA)
   )
 ENDIF ()
 
+IF (KOKKOS_ENABLE_SYCL)
+  KOKKOSKERNELS_INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR}/sycl)
+  KOKKOSKERNELS_INCLUDE_DIRECTORIES(REQUIRED_DURING_INSTALLATION_TESTING ${CMAKE_CURRENT_SOURCE_DIR}/sycl)
+
+  APPEND_GLOB(SYCL_BLAS_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/sycl/Test_SYCL_Blas*.cpp)
```
Comment (lucbv) [inline unit_test/CMakeLists.txt:94]: There is only one source file in SYCL: `Test_SYCL_Blas.cpp` so I think the APPEND_GLOB should be removed and the `${SYCL_BLAS_SOURCES}` should be replaced with `sycl/Test_SYCL_Blas.cpp`.
This is also the idea behind the refactor of the unit-test, this way developers are using header only tests that automatically get propagated to all the backends.

File: unit_test/CMakeLists.txt [hunk 2]
```diff
@@ -87,6 +87,58 @@ IF (KOKKOS_ENABLE_CUDA)
...
+  KOKKOSKERNELS_ADD_UNIT_TEST(
+    blas_sycl
+    SOURCES
+      Test_Main.cpp
+      ${SYCL_BLAS_SOURCES}
+    COMPONENTS blas
+    )
+
+  APPEND_GLOB(SYCL_BATCHED_DLA_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/sycl/Test_SYCL_Batched*.cpp)
```
Comment (lucbv) [inline unit_test/CMakeLists.txt:103]: Same as above

File: unit_test/CMakeLists.txt [hunk 3]
```diff
@@ -87,6 +87,58 @@ IF (KOKKOS_ENABLE_CUDA)
...
+  KOKKOSKERNELS_ADD_UNIT_TEST(
+    batched_dla_sycl
+    SOURCES
+      Test_Main.cpp
+      ${SYCL_BATCHED_DLA_SOURCES}
+    COMPONENTS batched
+    )
+
+  # FIXME_SYCL
+  #APPEND_GLOB(SYCL_SPARSE_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/sycl/Test_SYCL_Sparse*.cpp)
```
Comment (lucbv) [inline unit_test/CMakeLists.txt:113]: Same as above

File: unit_test/CMakeLists.txt [hunk 4]
```diff
@@ -87,6 +87,58 @@ IF (KOKKOS_ENABLE_CUDA)
...
+  #KOKKOSKERNELS_ADD_UNIT_TEST(
+  #  sparse_sycl
+  #  SOURCES
+  #    Test_Main.cpp
+  #    ${SYCL_SPARSE_SOURCES}
+  #  COMPONENTS sparse
+  #)
+
+  APPEND_GLOB(SYCL_GRAPH_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/sycl/Test_SYCL_Graph*.cpp)
```
Comment (lucbv) [inline unit_test/CMakeLists.txt:122]: Same as above

File: unit_test/CMakeLists.txt [hunk 5]
```diff
@@ -87,6 +87,58 @@ IF (KOKKOS_ENABLE_CUDA)
...
+    KOKKOSKERNELS_ADD_UNIT_TEST(
+    graph_sycl
+    SOURCES
+      Test_Main.cpp
+      ${SYCL_GRAPH_SOURCES}
+    COMPONENTS graph
+  )
+
+  # FIXME_SYCL currently float 128 test is not working. So common tests are explicitly added.
+  APPEND_GLOB(SYCL_COMMON_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/sycl/Test_SYCL_Common*.cpp)
```
Comment (lucbv) [inline unit_test/CMakeLists.txt:132]: Same as above

File: unit_test/sycl/Test_SYCL_Batched.cpp [hunk 6]
```diff
@@ -0,0 +1,9 @@
+#ifndef TEST_SYCL_BATCHED_CPP
+#define TEST_SYCL_BATCHED_CPP
+#define KOKKOSKERNELS_SYCL_BATCHED_TESTS
```
Comment (lucbv) [inline unit_test/sycl/Test_SYCL_Batched.cpp:3]: I am suspecting that this variable is not used, I hope I did not forget to remove it from the other backends...

**Review by lucbv — CHANGES_REQUESTED** [review]
This is fine in general but I do no like the idea of disabling some of the TeamVector tests without having a tracking mechanism. 
You can disable the whole batched_sycl test, people will clearly know that there is an issue in there...
You can also open an issue for that but I am not sure if this the safest approach?

File: unit_test/sycl/Test_SYCL_Batched.cpp (hunk 6 above)
Comment (masterleinad) [reply unit_test/sycl/Test_SYCL_Batched.cpp:3]: It's also in `CUDA` where I copied it from. :slightly_smiling_face:

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pre-Test Inspection' - - This Pull Request Requires Inspection... The code must be inspected by a member of the Team before Testing/Merging
           THE LAST COMMIT TO THIS PULL REQUEST HAS BEEN REVIEWED, BUT NOT ACCEPTED OR REQUIRES CHANGES

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pre-Test Inspection' - - This Pull Request Requires Inspection... The code must be inspected by a member of the Team before Testing/Merging
           THE LAST COMMIT TO THIS PULL REQUEST HAS NOT BEEN REVIEWED YET!

**Comment by masterleinad** [comment]
> This is fine in general but I do no like the idea of disabling some of the TeamVector tests without having a tracking mechanism.
> You can disable the whole batched_sycl test, people will clearly know that there is an issue in there...
> You can also open an issue for that but I am not sure if this the safest approach?

If I disable all categories in which failing tests appear, then I'm left with `common_sycl`. You can always find tests that are problematic by searching for `FIXME_SYCL`.
I mostly care about having the test infrastructure in place. Getting all the tests passing can be done incrementally IMHO. That being said just let me know what you want me to do.

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pull Request AutoTester' - User Requested Retest - Label AT: RETEST will be reset after testing.

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pull Request AutoTester' - User Requested Retest - Label AT: RETEST will be reset after testing.

**Review by lucbv — APPROVED** [review]
OK, I will open an issue with the list of tests failing but that should be fine

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pre-Test Inspection' - SUCCESS: The last commit to this Pull Request has been INSPECTED AND APPROVED by [ lucbv ]!

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pull Request AutoTester' - Testing Jenkins Projects:

<details>
<summary>
  <b>Pull Request Auto Testing STARTING</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC720</h3>
<ul>
  <li>Build Num: 194</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>sycl_add_tests</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/masterleinad/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>315327dc2c98f2870873bad4f6c3bb6f49f32bf7</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>90970cc1a5d67369eb4f8ad5188f80880aab511d</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>927</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_GCC720</h3>
<ul>
  <li>Build Num: 187</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>sycl_add_tests</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/masterleinad/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>315327dc2c98f2870873bad4f6c3bb6f49f32bf7</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>90970cc1a5d67369eb4f8ad5188f80880aab511d</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>927</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_INTEL18</h3>
<ul>
  <li>Build Num: 173</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>sycl_add_tests</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/masterleinad/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>315327dc2c98f2870873bad4f6c3bb6f49f32bf7</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>90970cc1a5d67369eb4f8ad5188f80880aab511d</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>927</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC720_Light</h3>
<ul>
  <li>Build Num: 207</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>sycl_add_tests</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/masterleinad/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>315327dc2c98f2870873bad4f6c3bb6f49f32bf7</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>90970cc1a5d67369eb4f8ad5188f80880aab511d</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>927</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_CUDA10</h3>
<ul>
  <li>Build Num: 171</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>sycl_add_tests</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/masterleinad/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>315327dc2c98f2870873bad4f6c3bb6f49f32bf7</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>90970cc1a5d67369eb4f8ad5188f80880aab511d</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>927</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_CUDA9</h3>
<ul>
  <li>Build Num: 168</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>sycl_add_tests</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/masterleinad/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>315327dc2c98f2870873bad4f6c3bb6f49f32bf7</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>90970cc1a5d67369eb4f8ad5188f80880aab511d</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>927</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_GCC720_GCC740</h3>
<ul>
  <li>Build Num: 166</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>sycl_add_tests</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/masterleinad/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>315327dc2c98f2870873bad4f6c3bb6f49f32bf7</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>90970cc1a5d67369eb4f8ad5188f80880aab511d</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>927</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Using Repos:</h2>

<dl>
  <dt>Repo: KOKKOSKERNELS (masterleinad/kokkos-kernels)</dt>
  <ul>
    <li>Branch: sycl_add_tests</li>
    <li>SHA: 315327dc2c98f2870873bad4f6c3bb6f49f32bf7</li>
    <li>Mode: TEST_REPO</li>
  </ul>
</dl>

Pull Request Author: masterleinad
</details>

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pull Request AutoTester' - Jenkins Testing: all Jobs PASSED

<details>
<summary>
  <b>Pull Request Auto Testing has PASSED</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC720</h3>
<ul>
  <li>Build Num: 194</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>sycl_add_tests</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/masterleinad/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>315327dc2c98f2870873bad4f6c3bb6f49f32bf7</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>90970cc1a5d67369eb4f8ad5188f80880aab511d</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>927</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_GCC720</h3>
<ul>
  <li>Build Num: 187</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>sycl_add_tests</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/masterleinad/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>315327dc2c98f2870873bad4f6c3bb6f49f32bf7</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>90970cc1a5d67369eb4f8ad5188f80880aab511d</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>927</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_INTEL18</h3>
<ul>
  <li>Build Num: 173</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>sycl_add_tests</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/masterleinad/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>315327dc2c98f2870873bad4f6c3bb6f49f32bf7</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>90970cc1a5d67369eb4f8ad5188f80880aab511d</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>927</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_GCC720_Light</h3>
<ul>
  <li>Build Num: 207</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>sycl_add_tests</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/masterleinad/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>315327dc2c98f2870873bad4f6c3bb6f49f32bf7</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>90970cc1a5d67369eb4f8ad5188f80880aab511d</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>927</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_CUDA10</h3>
<ul>
  <li>Build Num: 171</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>sycl_add_tests</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/masterleinad/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>315327dc2c98f2870873bad4f6c3bb6f49f32bf7</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>90970cc1a5d67369eb4f8ad5188f80880aab511d</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>927</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_CUDA9</h3>
<ul>
  <li>Build Num: 168</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>sycl_add_tests</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/masterleinad/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>315327dc2c98f2870873bad4f6c3bb6f49f32bf7</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>90970cc1a5d67369eb4f8ad5188f80880aab511d</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>927</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: KokkosKernels_PullRequest_Tpls_GCC720_GCC740</h3>
<ul>
  <li>Build Num: 166</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_BRANCH</td>
    <td>sycl_add_tests</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_REPO</td>
    <td>https://github.com/masterleinad/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_SOURCE_SHA</td>
    <td>315327dc2c98f2870873bad4f6c3bb6f49f32bf7</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_REPO</td>
    <td>https://github.com/kokkos/kokkos-kernels</td>
  </tr>

  <tr>
    <td>KOKKOSKERNELS_TARGET_SHA</td>
    <td>90970cc1a5d67369eb4f8ad5188f80880aab511d</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>927</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>KOKKOSKERNELS</td>
  </tr>
</table>
</details>

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pre-Merge Inspection' - SUCCESS: The last commit to this Pull Request has been INSPECTED AND APPROVED by [ lucbv ]!

**Comment by kokkos-devops-admin** [comment]
Status Flag 'Pull Request AutoTester' - Pull Request MUST BE MERGED MANUALLY BY Project Team - This Repo does not support Automerge

**Comment by masterleinad** [comment]
We will need this one for https://github.com/trilinos/Trilinos/pull/9086.

**Pull request merged**